packages/flutter_tools/test/android/android_workflow_test.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:file/memory.dart';
+import 'package:flutter_tools/src/base/file_system.dart';
+import 'package:flutter_tools/src/base/io.dart';
+import 'package:flutter_tools/src/android/android_sdk.dart';
+import 'package:flutter_tools/src/android/android_workflow.dart';
+import 'package:mockito/mockito.dart';
+import 'package:process/process.dart';
+import 'package:test/test.dart';
+
+import '../src/context.dart';
+import '../src/mocks.dart' show MockProcessManager, MockStdio;
+
+void main() {
+  AndroidSdk sdk;
+  MemoryFileSystem fs;
+  MockProcessManager processManager;
+  MockStdio stdio;
+
+  setUp(() {
+    sdk = new MockAndroidSdk();
+    fs = new MemoryFileSystem();
+    processManager = new MockProcessManager();
+    stdio = new MockStdio();
+  });
+
+  testUsingContext('FOo', () async {
```

Comment:
nit: maybe a better name for the test?

Reply:
Probably looking at an older version -- there are 3 tests now.